### ISSUE TITLE
This 'while' statement doesn't loop

### ISSUE THREAD

**Issue by HelgeStenstrom** [goxr3plus/XR3Player#29]
https://github.com/goxr3plus/XR3Player/blob/7d79f3b4ccca604a34e22e593b2510710580303c/src/main/java/com/goxr3plus/xr3player/services/xplayer/XPlayerPlayService.java#L340-L343

This 'while' statement doesn't loop. Does it work as intended? Can it be replaces with an if() structure instead?

**Comment by goxr3plus** [comment]
Yeah, it checks if the entry exists in the database that means if resultset has at least one element, i think it can be replaced with an if statement :)

**Comment by goxr3plus** [comment]
Good catch, you arw doing amazing work i am impressed.

**Comment by goxr3plus** [comment]
@HelgeStenstrom  Here is the solution :

https://javarevisited.blogspot.com/2016/10/how-to-check-if-resultset-is-empty-in-Java-JDBC.html

**Comment by goxr3plus** [comment]
@HelgeStenstrom  How you can do that ? Sending me a link with the exact line of the file in github like you did ? 

I don't know how to do it ... can you show me ?

**Comment by goxr3plus** [comment]
Oouuu i found out :) => https://help.github.com/en/articles/creating-a-permanent-link-to-a-code-snippet

**Comment by HelgeStenstrom** [comment]
Most of my findings are from IntelliJ’s code analysis.

**Comment by goxr3plus** [comment]
Would you like to solve thid issue here is a solution?

https://javarevisited.blogspot.com/2016/10/how-to-check-if-resultset-is-empty-in-Java-JDBC.html

IT'S easy for me but in case you want to give it a try :)

**Comment by HelgeStenstrom** [comment]
I probably can’t do any coding until August. I leave this to you.
